internal/strace/strace.go

```diff
@@ -129,6 +141,17 @@ func (r *Result) recordFileAccess(file string, read, write, delete bool) {
 	r.files[file].Delete = r.files[file].Delete || delete
 }
 
+func (r *Result) recordFileWrite(file string, bytesWritten int64) {
+	if bytesWritten > 0 {
+		if _, exists := r.files[file]; !exists {
```

Comment:
Can we just call `recordFileAccess` here?

Also, no need to initialize `writeInfo`. `nil` is a valid slice value to append to.